brave/brave-core
# Disable motion sensors and background sync by default

Fixes https://github.com/brave/brave-browser/issues/4709 and https://github.com/brave/brave-browser/issues/4789.

## Submitter Checklist:

- Verified that these changes build without errors on
- Verified that these changes pass automated tests (unit, browser, security tests) on

## Test Plan:

For background sync:

1. Load https://deanhume.github.io/Service-Workers-BackgroundSync/
2. Open the devtools console.
3. Press the "Make an HTTP request - do it!" button.
4. Look for "Unable to fetch image." in the console (and not "sync registered").

For either feature:

1. Load https://brave.com
2. Click the padlock icon in the URL bar.
3. Click "Site settings".
4. Look for "Motion sensors" and "Background sync". They should both be labeled "Block (default)".

## Reviewer Checklist:


## After-merge Checklist:

  changes has landed on.
  - [ ] Update https://github.com/brave/brave-browser/wiki/Deviations-from-Chromium-(features-we-disable-or-remove).
